iOS/DuckDuckGo/AppLifecycle/AppDependencyProvider.swift

```diff
@@ -117,7 +117,14 @@ final class AppDependencyProvider: DependencyProvider {
         var authenticationStateProvider: (any SubscriptionAuthenticationStateProvider)!
 
         let tokenStorageV2 = SubscriptionTokenKeychainStorageV2(keychainType: .dataProtection(.named(subscriptionAppGroup))) { keychainType, error in
-            Pixel.fire(.privacyProKeychainAccessError, withAdditionalParameters: ["type": keychainType.rawValue, "error": error.errorDescription])
+
+            let parameters = [PixelParameters.privacyProKeychainAccessType: keychainType.rawValue,
+                              PixelParameters.privacyProKeychainError: error.localizedDescription,
```

Comment:
As discussed on MM we should use `errorDescription` here.

Reply:
Fixed everywhere we know the error type